IDisposableExercise/IDisposableExercise/Student.cs

```diff
@@ -0,0 +1,21 @@
+﻿using Microsoft.Win32.SafeHandles;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Runtime.InteropServices;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace IDisposableExercise
+{
+    /// <summary>
+    /// Represents a student person.
+    /// </summary>
+    public class Student : Person
```

Comment:
It would have been good to include an override of Dispose here to demonstrate the full scenario with a derived class. (-0.1)